public/sass/components/_gf-form.scss

```diff
@@ -90,6 +90,10 @@ $input-border: 1px solid $input-border-color;
 }
 
 .gf-form-button-row {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-content: flex-start;
```

Comment:
please remove these and make the or new React Button an inline-flex instead, that is what did in your other branch, I think a Button should be an inline element, what do you think? and @dprokop?

Reply:
Yes, it should be an inline-block.